README.md

```diff
@@ -1 +1,311 @@
-# builder
+[![Travis Status][trav_img]][trav_site]
+
+Builder
+=======
+
+Builder is a task runner.
+
+Builder is an enhancement to `npm run TASK`.
+
+Builder is a meta-tool for all your common build, quality, and test tasks.
+
+## Overview
+
+At a high level `builder` is a tool for consuming `package.json` `scripts`
+commands, providing sensible / flexible defaults, and support various scenarios
+("archetypes") for your common use cases across multiple projects.
+
+Builder is not opinionated, although archetypes _are_ and typically dictate
+file structure, standard configurations, and dev workflows. Builder supports
+this in an agnostic way, providing essentially the following:
+
+* `NODE_PATH`, `PATH` enhancements to run, build, import from archetypes so
+  dependencies and configurations don't have to be installed directly in a
+  root project.
+* A task runner capable of single tasks (`run`) or multiple concurrent tasks
+  (`concurrent`).
+* An intelligent merging of `package.json` `scripts` tasks.
+
+... and that's about it!
+
+### Usage
+
+To start using builder, install and save `builder` and any archetypes you
+intend to use. We'll use the [builder-react-component][] archetype as an
+example.
+
+#### Global Install
+
+For ease of use, one option is to globally install `builder` and locally install
+archetypes:
+
+```sh
+$ npm install -g builder
+$ npm install --save builder-react-component
+```
+
+Like a global install of _any_ Node.js meta / task runner tool (e.g., `eslint`,
+`mocha`, `gulp`, `grunt`) doing a global install is painful because:
+
+* You are tied to _just one_ version of the tool for all projects.
+* You must also globally install the tool in CI, on servers, etc.
+
+... so instead, we **strongly recommend** a local install described in the
+next section!
```

Comment:
@divmain @chaseadamsio -- I've added this via https://github.com/FormidableLabs/builder/commit/5a9eb02722d27b37a143fa6aa5c6e7a84f88703a 

Basically where I'm at is "this is a fundamental **NODE** thing". Let's not try and solve how node js generally does things on this PR. I'll leave a strong recommendation above here to inform folks what they should already know.

We can take up additional discussion in a ticket ;)